src/main/java/uk/gov/companieshouse/company_appointments/CompanyAppointmentController.java

```diff
@@ -6,12 +6,14 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import uk.gov.companieshouse.company_appointments.model.view.AllCompanyAppointmentsView;
 import uk.gov.companieshouse.company_appointments.model.view.CompanyAppointmentView;
 import uk.gov.companieshouse.logging.Logger;
 import uk.gov.companieshouse.logging.LoggerFactory;
 
 @Controller
-@RequestMapping(path = "/company/{company_number}/appointments/{appointment_id}", produces = "application/json")
+@RequestMapping(path = "/company/{company_number}/", produces = "application/json")
```

Comment:
I think the final slash here should be in the below paths so 
```suggestion
@RequestMapping(path = "/company/{company_number}", produces = "application/json")
```